scripts/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+if [ $? != 0 ]; then
+  printf "\n\n$Red$(echo Build failed.)$Color_Off"
+  exit 1
+fi
+
+tsc -p tsconfig.build.json
+
+if [ $? != 0 ]; then
+  printf "\n\n$Red$(echo Build failed.)$Color_Off"
+  exit 1
+fi
+
+npx tscpaths -p tsconfig.json -s src -o dist
+
+if [ $? != 0 ]; then
+  printf "\n\n$Red$(echo Build failed.)$Color_Off"
+  exit 1
+fi
+
+yarn sass src/styles.scss dist/styles.css --no-source-map
```

Comment:
Why are you using `npx` for the ts -> js translation, but yarn for sass? 

Do you use yarn for everything you do with this repo? I really don't have a good understanding of the differences between yarn and npm and why/where one would use yarn. Any context you have would be helpful!